### ISSUE TITLE
Support TLS

### ISSUE THREAD

**Issue by Lukasa** [kespindler/albatross#2]
HTTPS is great, and asyncio makes it pretty easy to have TLS support built-in with pretty good configuration.

**Comment by kespindler** [comment]
Is this as simple as passing ssl=SSLContext(...) to the start_server method?

**Comment by Lukasa** [comment]
@kespindler As far as I can see from a quick spin over the documentation, it seems to be. You should use `create_default_context`, of course, but otherwise that seems to be the whole requirement.

The bigger problem is that you need a cert chain to present. That's a legitimate reason to want to hold off on doing this immediately: you need a way for your users to tell you where the cert chain and private key are!

**Comment by kespindler** [comment]
Yeah. Production systems should have ssl termination at the load balancer _grumble grumble_

I'll think about this
